packages/block-library/src/post-content/edit.js

```diff
@@ -147,7 +152,10 @@ export default function PostContentEdit( {
 	return (
 		<RecursionProvider uniqueId={ contextPostId }>
 			{ contextPostId && contextPostType ? (
-				<Content context={ context } layout={ layout } />
```

Comment:
Note: the removal of passing down `layout` is safe, as the usage of the passed down prop was removed in https://github.com/WordPress/gutenberg/pull/47477 — for the `EditableContent` state of the Post Content block, the layout classnames are already handled by `useInnerBlocksProps`.